packages/framework/tinylicious-client/src/TinyliciousClient.ts

```diff
@@ -115,11 +115,12 @@ export class TinyliciousClient {
         };
     }
 
-    private createLoader(containerSchema: ContainerSchema) {
+    private createLoader(containerSchema: ContainerSchema): Loader {
         const containerRuntimeFactory = new DOProviderContainerRuntimeFactory(
             containerSchema,
         );
         const module = { fluidExport: containerRuntimeFactory };
+        // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
         const codeLoader = { load: async () => module };
```

Comment:
Are there types we can use here?

Reply:
Would `allowExpressions: true` ameliorate this case? I'm not in favor of having to annotate stuff like this, where TypeScript will infer the type anyway and that inferred type will be used to validate its usage below.

Reply:
Made a note here: https://github.com/microsoft/FluidFramework/pull/9502/files#r835464769